core/src/com/unciv/ui/utils/CameraStageBaseScreen.kt

```diff
@@ -60,10 +67,19 @@ open class CameraStageBaseScreen : Screen {
 
     override fun dispose() {}
 
-    fun displayTutorials(name: String) {
-        tutorials.displayTutorials(name,this)
+    /**
+     * Call this if you want to show tutorials on screen
+     */
+    fun initTutorial(screen: CameraStageBaseScreen) {
```

Comment:
Why not init the tutorials on every screen by default? It's just creating classes, not textures or strings etc.
This will make a lot of the other code a lot more readable as well.

Reply:
because `CameraStageBaseScreen` is and open class and we can get a leak here

Reply:
Oh, and civilopesia screen doesn't need tutorial

Reply:
I have an idea how to manage this, will check in Thursday